assets/monero.conf.template

```diff
@@ -1,41 +1,41 @@
 #
 # monero.conf configuration file.  Things beginning with # are comments.
 #
-# See https://monerodocs.org/interacting/monerod-reference/
+# See https://getmonero.dev/interacting/monerod
```

Comment:
```suggestion
# See https://docs.getmonero.org/interacting/monerod
```

soon ^ 🙃

Reply:
Good to know it's coming but we definitely will go with a working link over a broken one.

Reply:
Its live now

Reply:
![image](https://github.com/user-attachments/assets/afb9281f-d914-43b0-9e89-53dfacb94f16)

Reply:
Ah, I see, it's at https://docs.getmonero.org/interacting/monerod-reference/
```suggestion
# See https://docs.getmonero.org/interacting/monerod-reference/
```